packages/core/src/components/button/_common.scss

```diff
@@ -64,19 +64,23 @@ $button-gradient: linear-gradient(to bottom, rgba($white, 0.8), rgba($white, 0))
 $button-intent-gradient: linear-gradient(to bottom, rgba($white, 0.1), rgba($white, 0)) !default;
 $dark-button-gradient: linear-gradient(to bottom, rgba($white, 0.05), rgba($white, 0)) !default;
 
+$button-color-placeholder: $pt-text-color-muted !default;
```

Comment:
I think we should use `$pt-text-color-disabled` here. The difference with `-muted` is so small that I might be unnoticeable in some cases

Reply:
how about `$input-placeholder-color: $pt-text-color-disabled !default;`?

also, one of these names is wrong? should be `button-placeholder-color`. https://github.com/palantir/blueprint/wiki/Coding-guidelines#sass

Reply:
Good catch with `button-placeholder-color`

Reply:
`$button-placeholder-color: $pt-text-color-disabled !default;`